src/main/java/frc/robot/utils/MovingAverage.java

```diff
@@ -0,0 +1,103 @@
+package frc.robot.utils;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Map;
+import java.util.TreeMap;
+
+/**
+ * The moving average contains a list of velocity based on the distance, and should give an approximation of the velocity to apply.
+ * In order to the moving average object to approximate better, you need to pass more values.
+ * The object calculate calculate a strait line between the closest point, and retrieve the velocity (in the strait line between the records)
+ */
+public class MovingAverage {
```

Comment:
We don't need a csv file for the moving average, we just need an array of certain distances and an array of the corresponding velocities

Reply:
It's the same, just using the csv file is easier because we will just need to save the file, and then gradle will transform it into the jar. we won't have to copy the values manually every time we update our data

Reply:
OK